### PR TITLE
If e2e app is disabled, show info

### LINKED ISSUE #2242: Check if E2E is still available
- fresh installation
- have encrypted folders in account
- disable e2e app
- click on encrypted folder
-> tries to generate & upload keys

Instead app should warn and do not enter folder.
(This should not happen and if it is a "fault" of the admin, but let us be safe here)

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Fixes #2242

- fresh installation
- have encrypted folders in account
- disable e2e app
- click on encrypted folder
-> warning as snackbar: End to end encryption not enabled on server.

Signed-off-by: tobiaskaminsky <tobias@kaminsky.me>

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
:+1: 

two small remarks:
* remove unused imports please
* I'd rephrase the message to: "End to end encryption disabled on server." since negation are harder to comprehend

**Comment by tobiasKaminsky** [comment]
Imports should automatically cleaned by AndroidStudio...
Changed the wording, :+1:

**Comment by mario** [comment]
👍

**Comment by AndyScherzinger** [comment]
:+1: 
@tobiasKaminsky can you change it to the master Snapshot :)
